source/catalog.html

```diff
@@ -183,24 +183,24 @@ <h3 class="card__title title">Cat Energy SLIM 500 г</h3>
           <a class="card__button button" href="#">Заказать</a>
         </li>
       </ul>
-      <div class="products__showmore">
-        <h3 class="products__title title">Показать еще 100500 товаров</h3>
-        <p class="products__description">На самом деле вкусов гораздо больше!</p>
-        <a class="products__button-showmore button" href="#">Показать все</a>
+      <div class="card card--showmore">
+        <h3 class="card__title--showmore">Показать еще 100500 товаров</h3>
```

Comment:
```suggestion
        <h3 class="card__title card__title--showmore">Показать еще 100500 товаров</h3>
```
Модификатор не используется в одиночку